ResearchKit/Localized/en.lproj/ResearchKit.strings

```diff
@@ -341,27 +341,31 @@
+"TREMOR_TEST_INTRO_2_RIGHT_HAND_TEXT" = "Hold the phone in your right hand as shown in the image below.";
+"TREMOR_TEST_INTRO_2_LEFT_HAND_TEXT" = "Hold the phone in your left hand as shown in the image below.";
+"TREMOR_TEST_INTRO_2_DETAIL_DEFAULT_%" = "You will be asked to perform %@ while sitting with the phone in your hand.";
+"TREMOR_TEST_INTRO_2_DETAIL_BOTH_HANDS_%" = "You will be asked to perform %@ while sitting with the phone first in one hand, then again in the other.";
 "TREMOR_TEST_INTRO_2_DETAIL_1_TASK" = "a task";
 "TREMOR_TEST_INTRO_2_DETAIL_2_TASK" = "two tasks";
 "TREMOR_TEST_INTRO_2_DETAIL_3_TASK" = "three tasks";
 "TREMOR_TEST_INTRO_2_DETAIL_4_TASK" = "four tasks";
 "TREMOR_TEST_INTRO_2_DETAIL_5_TASK" = "five tasks";
 "TREMOR_TEST_ACTIVE_STEP_INTRO_TEXT" = "Tap next to proceed.";
-"TREMOR_TEST_ACTIVE_STEP_IN_LAP_INTRO" = "Prepare to hold your phone in your lap";
-"TREMOR_TEST_ACTIVE_STEP_IN_LAP_INSTRUCTION_%ld" = "Keep holding your phone in your lap for %ld seconds";
-"TREMOR_TEST_ACTIVE_STEP_EXTEND_ARM_INTRO" = "Now hold phone with hand extended out at shoulder height";
-"TREMOR_TEST_ACTIVE_STEP_EXTEND_ARM_INSTRUCTION_%ld" = "Keep holding phone with hand extended for %ld seconds";
-"TREMOR_TEST_ACTIVE_STEP_BEND_ARM_INTRO" = "Now hold phone at shoulder height with elbow bent";
+"TREMOR_TEST_ACTIVE_STEP_IN_LAP_INTRO" = "Prepare to hold your phone in your lap.";
+"TREMOR_TEST_ACTIVE_STEP_IN_LAP_INSTRUCTION_%ld" = "Keep holding your phone in your lap for %ld seconds.";
+"TREMOR_TEST_ACTIVE_STEP_EXTEND_ARM_INTRO" = "Now hold phone with hand extended out at shoulder height.";
+"TREMOR_TEST_ACTIVE_STEP_EXTEND_ARM_INSTRUCTION_%ld" = "Keep holding phone with hand extended for %ld seconds.";
+"TREMOR_TEST_ACTIVE_STEP_BEND_ARM_INTRO" = "Now hold phone at shoulder height with elbow bent.";
 "TREMOR_TEST_ACTIVE_STEP_BEND_ARM_INSTRUCTION_%ld" = "Keep holding phone with elbow bent for %ld seconds";
-"TREMOR_TEST_ACTIVE_STEP_TOUCH_NOSE_INTRO" = "Now keeping your elbow bent, touch phone to nose repeatedly";
+"TREMOR_TEST_ACTIVE_STEP_TOUCH_NOSE_INTRO" = "Now keeping your elbow bent, touch phone to nose repeatedly.";
 "TREMOR_TEST_ACTIVE_STEP_TOUCH_NOSE_INSTRUCTION_%ld" = "Keep touching phone to nose for %ld seconds";
-"TREMOR_TEST_ACTIVE_STEP_TURN_WRIST_INTRO" = "Prepare to do a queen wave (wave by turning your wrist)";
-"TREMOR_TEST_ACTIVE_STEP_TURN_WRIST_INSTRUCTION_%ld" = "Keep performing a queen wave for %ld seconds";
-"TREMOR_TEST_ACTIVE_STEP_FINISHED_INSTRUCTION" = "Continue to next task";
-"TREMOR_TEST_COMPLETED_INSTRUCTION" = "Activity completed";
+"TREMOR_TEST_ACTIVE_STEP_TURN_WRIST_INTRO" = "Prepare to do a queen wave (wave by turning your wrist).";
+"TREMOR_TEST_ACTIVE_STEP_TURN_WRIST_INSTRUCTION_%ld" = "Keep performing a queen wave for %ld seconds.";
+"TREMOR_TEST_ACTIVE_STEP_SWITCH_HANDS_INSTRUCTION" = "Now switch the phone to your other hand and continue to the next task.";
```

Comment:
Ideally, I would like for this to have left/right switching instructions. Nice and explicit:

"Now switch the phone to your left hand and continue to the next task."
"Now switch the phone to your right hand and continue to the next task."